proto/errorpb.proto

```diff
@@ -12,47 +12,77 @@ option (rustproto.lite_runtime_all) = true;
 
 option java_package = "org.tikv.kvproto";
 
+// NotLeader is the error variant that tells a request be handle by raft leader 
+// is sent to raft follower.
 message NotLeader {
+    // the current region ID
     uint64 region_id = 1;
+    // the current region leader
     metapb.Peer leader = 2;
 }
 
+// StoreNotMatch is the error variant that tells the request is sent to wrong 
+// store. (i.e. inconsistency of the store ID that request shows and the real store ID.)
 message StoreNotMatch {
+    // store id in request
     uint64 request_store_id = 1;
+    // real store id
     uint64 actual_store_id = 2;
 }
 
+// RegionNotFound is the error variant that tells there isn't any region matches 
+// need of the request.
 message RegionNotFound {
+    // the current region ID
     uint64 region_id = 1;
 }
 
+// KeyNotInRegion is the error variant that tells the key the request requires isn't present in
+// this region. 
 message KeyNotInRegion {
+    // the requested key
     bytes key = 1;
+    // the current region ID
     uint64 region_id = 2;
+    // the current region start key
     bytes start_key = 3;
+    // the current region end key
     bytes end_key = 4;
 }
 
+// KeyNotInRegion is the error variant that tells the region metainfo is updated. 
```

Comment:
What's the `metainfo`?

Reply:
I wonder... Is there any other event will update epoch expect splitting / merging?

(e.g. since PD runs raft internally, does membership change can be done by update epoch in PD, instead of commit a log entry at `raft_rs`?)

Reply:
Raft Confchange also changes epoch.